docs(srv): split troubleshootings into multiple pages MTA-5198 (#3955)
scaleway/docs-content: serverless/functions/troubleshooting/timeout-error.mdx

---
meta:
  title: I am experiencing timeout errors
  description: Troubleshoot timeout errors when using Scaleway Serverless Functions.
content:
  h1: I am experiencing timeout errors
  paragraph: Troubleshoot timeout errors when using Scaleway Serverless Functions.
tags: serverless functions troubleshooting issue error function timeout
dates:
  validation: 2024-11-06
  posted: 2024-11-06
categories:
  - serverless
---

## Problem

I am experiencing timeout errors when using Scaleway Serverless Functions.

### Cause

> Timeout happens when your function can’t return a results in the defined amount of time.

Reviewer: ```suggestion
Timeout happens when your function cannot return any result in the defined amount of time.
```
